### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by whoevertf**

### CODE REVIEW

File: src/routes/__layout.svelte [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <div class="app-container">
 	<div id="logo-container">
-		<a href="{base}/" id="logo">JobFinder</a>
+		<!-- <a href="{base}/" id="logo">JobFinder</a> -->
 	</div>
```
Comment (timolai-andrievich) [inline src/routes/__layout.svelte:8]: Please delete the `logo-container` as it is not used

File: src/routes/__layout.svelte [hunk 2]
```diff
@@ -35,5 +35,5 @@
 		display: flex;
 		flex-direction: row;
 		justify-content: end;
-	}
+	} */
```
Comment (timolai-andrievich) [inline src/routes/__layout.svelte:38]: Please delete commented selectors, as we are using GIt, there is no point in keeping dead code.

File: src/routes/hot.svelte [hunk 3]
```diff
@@ -18,14 +18,19 @@
...
+		<header>
+			<img src="{base}/fire.png" alt="Fire logo" />
+			<h1>List of the hottest<br /> vacancies so far.</h1>
+		</header>
+		<div class="vacancies-list" id="vacancies-list">
+			{#each vacancies as vacancy}
+				<Vacancy data={vacancy} />
+			{/each}
+		</div>
 	</div>
```
Comment (timolai-andrievich) [inline src/routes/hot.svelte:34]: `.app-container` selector is useless, as whole page is already wrapped in `<main>`

File: src/routes/index.svelte [hunk 4]
```diff
@@ -2,4 +2,168 @@
...
+	<title>JobFinder - Homepage</title>
+</svelte:head>
+
+<div class="main-block">
+	<img src="{base}/logo_main.png" alt="JobFinder logo" />
+	<a href="{base}/filter"><button class="btn buton">Discover Jobs</button></a>
+</div>
+<div class="hot">
+	<div class="heading-hot">
+		<img src="{base}/fire copy.png" alt="fire icon" />
```
Comment (timolai-andrievich) [inline src/routes/index.svelte:15]: `fire_black.png` is a more descriptive name.

File: src/routes/index.svelte [hunk 5]
```diff
@@ -2,4 +2,168 @@
...
 </script>
 
-<a href="{base}/hot">Hot vacancies</a>
+<svelte:head>
+	<title>JobFinder - Homepage</title>
+</svelte:head>
+
+<div class="main-block">
+	<img src="{base}/logo_main.png" alt="JobFinder logo" />
+	<a href="{base}/filter"><button class="btn buton">Discover Jobs</button></a>
```
Comment (timolai-andrievich) [inline src/routes/index.svelte:11]: `.buton` selector is not used anywhere, except for this button, so `.discovery-button` would be a more descriptive name.

File: src/routes/index.svelte [hunk 6]
```diff
@@ -2,4 +2,168 @@
...
+		text-decoration: none;
+	}
+	@font-face {
+		font-family: 'Lato-bold';
+		src: url('../lib/fonts/Lato-Bold.ttf');
+	}
+	@font-face {
+		font-family: 'Lato';
+		src: url('../lib/fonts/Lato-Regular.ttf');
+	}
```
Comment (timolai-andrievich) [inline src/routes/index.svelte:65]: Fonts are already defined in `__layout.svelte`
```suggestion
```

File: src/routes/index.svelte [hunk 7]
```diff
@@ -2,4 +2,168 @@
...
+	.research {
+		grid-area: two;
+	}
+	.compare {
+		grid-area: three;
+	}
+	.apply {
+		grid-area: four;
+	}
```
Comment (timolai-andrievich) [inline src/routes/index.svelte:144]: As far as I understand, these selectors are useless
```suggestion
```

**Review by timolai-andrievich — CHANGES_REQUESTED** [review]
Good job, however, I suggest some minor changes. I also do not understand the purpose of `favicon copy.png` and `expand copy.png`

File: src/routes/hot.svelte (hunk 3 above)
Comment (whoevertf) [reply src/routes/hot.svelte:34]: If I remove `.app-container`, page `Hot vacancies` will have strange view.

**Review by timolai-andrievich — APPROVED** [review]

**Pull request merged**